source/app/controllers/index.rb

```diff
@@ -1,3 +1,31 @@
 get '/' do
-  erb :index
+  erb :welcome
 end
+
+get '/gameboard' do
+  user1_id = User.find_by_id(1).id
+  user2_id = User.find_by_id(2).id
+  erb :gameboard, locals: {user1_id: user1_id, user2_id: user2_id}
+end
+
+post '/game' do
+  player1_id = params[:player_1_id]
+  player2_id = params[:player_2_id]
+  token1 = params[:player_1_token_id]
+  token2 = params[:player_2_token_id]
+
+  token1_id = Token.find_by(name: token1)
+  token2_id = Token.find_by(name: token2)
+  current_game = RockPaperScissors.new(player1_id, player2_id, token1, token2)
+  results = current_game.play
+  game = Game.create!(player_1_id: player1_id, player_2_id: player2_id, token_1_id: token2_id,
```

Comment:
So there is a little disconnect between your idea of a game, and the way your controller logic works.

Your game is setup so that it records both the players, then the winner / loser. Why do we need to repeat that data?

If we just store the winner and the looser, or just the player_1 and player_2, we can remove 2 of the columns which will clean this up nice.